storage_file/models/storage_file.py

```diff
@@ -58,7 +60,11 @@ class StorageFile(models.Model):
     )
     mimetype = fields.Char("Mime Type", compute="_compute_extract_filename", store=True)
     data = fields.Binary(
-        help="Datas", inverse="_inverse_data", compute="_compute_data", store=False
+        help="Datas",
```

Comment:
as you are there.... :)
```suggestion
        help="Data",
```